artemis-server/src/main/java/org/apache/activemq/artemis/core/remoting/impl/netty/NettyAcceptor.java

```diff
@@ -843,4 +847,12 @@ private Throwable getRootCause(Throwable throwable) {
          return (list.size() < 2 ? throwable : list.get(list.size() - 1));
       }
    }
+
+   public static int getCommunicationTimeout() {
```

Comment:
as noted about should be configured by configuration per acceptor instance, these then are redundant.